<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <include file="$(find cartpole_description)/launch/main.launch"/>
    <rosparam command="load" file="$(find cartpole_tests)/config/cartpole_ruippeixotog_params.yaml" />
    <!-- Launch the training system -->
    <node pkg="cartpole_tests" name="cartpole3D_ruippeixotog" type="cartpole3D_ruippeixotog.py" output="screen"/>
    <node pkg="cartpole_tests" name="reward_chart" type="reward_chart.py" output="screen"/>
</launch>